lib/stack_master/template_compiler.rb

```diff
@@ -2,10 +2,10 @@ module StackMaster
   class TemplateCompiler
     TemplateCompilationFailed = Class.new(RuntimeError)
 
-    def self.compile(config, template_file_path)
+    def self.compile(config, template_file_path, compiler_options={})
```

Comment:
Minor niggle with the whitespace here since it seems out of place with the rest of the file.

`compiler_options={}` => `compiler_options = {}`